### DISCUSSION TITLE
Change replication level of old volumes

### DISCUSSION THREAD

**Discussion by Levalife** [seaweedfs/seaweedfs#3061] [Q&A] [answered]
It's not clear to me how can I change the replication level for all existing volumes or collections?

**Comment by chrislusf** [comment, marked as answer]
see https://github.com/chrislusf/seaweedfs/blob/master/unmaintained/change_superblock/change_superblock.go

**Reply by kmlebedev** [reply]
Maybe it's worth adding a replication change to the shell?  Since a very obvious operation

**Reply by chrislusf** [reply]
Too tricky on a live cluster.

**Reply by kmlebedev** [reply]
Maybe it's worth writing about the need to change replication in .dat files in volume.configure.replication, in what cases is it necessary?
